src/main/java/org/ballerinalang/command/util/Distribution.java

```diff
@@ -62,10 +48,6 @@ public void setVersion(String version) {
         this.version = version;
     }
 
-    public String getType() {
```

Comment:
@suganyasuven Why these getters and above overloaded constructors are removed?

Reply:
Those are not used anywhere.